dbt/models/churnado_subscriptions.sql

```diff
@@ -0,0 +1,41 @@
+select
+  s.id as subscription_id
+  , s.customer_id
+  , s.plan_id
+  , s.simplified_plan_id
+  , s.canceled_at
+  , s.created_at
+  , s.quantity
+  , s.trial_start_at
+  , s.trial_end_at
+  , u.id as user_id
+  , datediff(day, s.trial_start_at, s.trial_end_at) as trial_length
+  , case -- if subscription was active on Dec 31 and was created two months before, we can use it in the training set
+      when s.created_at < '2017-11-01' and (s.canceled_at is null or s.canceled_at > '2017-12-31')
+      then true
+      else false
+    end as is_training
```

Comment:
I used something similar in the other model but I think it doesn't make sense since all the features will be computed based on the "train" date (2017-12-31 for training and X for the validation dataset).

Reply:
Yep, we can leave this out as well.